adafruit_trellism4.py

```diff
@@ -228,7 +234,7 @@ class TrellisM4Express:
             current_press = pressed
     """
 
-    def __init__(self, rotation=0):
+    def __init__(self, rotation: int = 0) -> None:
```

Comment:
Use the same `Literal` as before here

Reply:
yeah, I considered doing those like that....guess I should trust my instincts sometimes....